packages/api/src/domain/medical/document-query.ts

```diff
@@ -21,12 +22,16 @@ export type ConvertResult = (typeof convertResult)[number];
 export const calculateConversionProgress = ({
   patient,
   convertResult,
+  requestId,
```

Comment:
Do we absolutely need to ask for the `requestId` as part of this function? We have access to `patient.data.documentQueryProgress`, so it seems we don't need to.

Reply:
good point